### PR TITLE
Removes get out of jail free card for captains breaking laser case.

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Disabling ID scan does the same thing without the need for an extra var

**Comment by KorPhaeron** [comment]
Also you can still flush yourself down disposals, hand tele out, or just keep a locker in the doorway when you break the case. It is a gun with unlimited ammo with your name literally on it that spawns in the same room as you, people are going to take it.

**Comment by AnturK** [comment]
Doesn't mean we need to make it easy.

**Comment by Jazaen** [comment]
Or ask AI.

**Comment by AnturK** [comment]
Asking AI is legit solution because it can then laugh at inept Captain.

**Comment by KorPhaeron** [comment]
For the theft objective I guess?

I just dont think there is a realistic code solution to what sounds like a roleplay problem unless you make the gun somehow undesirable (but then why steal it?)

**Comment by AnturK** [comment]
You might be right this isn't the way to go about this.

**Comment by KorPhaeron** [comment]
I mean this PR is still fine since at the very least it means your office is a wreck if you use my listed solutions, but I think most people will trade a redundant room for the best gun on station

**Comment by kevinz000** [comment]
tfw i trigger coderbus yet again

File: code/game/area/areas.dm [hunk 1]
```diff
@@ -282,6 +284,9 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 		//Lockdown airlocks
 		for(var/obj/machinery/door/DOOR in RA)
```
Comment (lzimann) [inline code/game/area/areas.dm]: This should just be typecasted as airlock. Unless I'm wrong and regular doors can be locked...

**Comment by ChangelingRain** [comment]
Still doesn't keep you from putting, like, a closed emergency locker in the way of a door so it can't close.

**Comment by BeeSting12** [comment]
So this removes ID scan? 👎 That's impossible to fix without an AI.

**Comment by XDTM** [comment]
we have an id scan wire

**Comment by octareenroon91** [comment]
Why does the captain still not have access to it though? It's not like it's on loan from the centcom museum or something.

**Comment by Shadowlight213** [comment]
Just do what goon did and make the gun a plastic toy that needs to be built for it to actually be useful. That fixes the problem of captain's wanting to grab it at round start.
We can say the syndicate got fooled and thought it was the real thing.

**Comment by Anonmare** [comment]
Maybe you could make it lack a firing pin? That makes it undesirable for most people since they have to do extra work to make it useful.

**Comment by PKPenguin321** [comment]
The issue at hand isn't the gun, it's bypassing the lockdown when you break the case. If you want to nerf the gun for no reason do it in a different pr

**Comment by BeeSting12** [comment]
We've had the gun for all these years and not once has anyone complained about it, why is it such a balance issue now?

**Comment by PKPenguin321** [comment]
It isn't, the issue at hand isn't the gun, it's bypassing the lockdown when you break the case

**Comment by lzimann** [comment]
I really dislike this, I don't think it's an issue at all.

**Comment by KorPhaeron** [comment]
We didnt have the alarm for a long time either.

**Comment by jughu** [comment]
Implying you couldnt bypass the lockdown for years anyways just put the hand tele not in the same room and redirect the disposal to caps bedroom if somoene then smashes the case without grabbing the hand tele litteraly next room then g o o d luck

**Comment by blobbernaut** [comment]
I'd leave it in my office if it was harder to steal. 

Make the case only breakable by deconstructing it,  or during red alert the captain can ID it open.

**Comment by Lumipharon** [comment]
Give the case a stasis field. Fluff wise it would keep the gun perfectly preserved (human leather is pretty shitty), with the added bonus of making it impossible to break the glass until you depower the room and wait for the field to power down, or some shit.

Sure, it won't stop the condom from looting it, but it would make it marginally more difficult to steal.

Bonus points: Make a plasteel shield drop down around the case if the shield powers down, that has to be blown off (c4 or some shit), or surgery drilled open.
That would make it slower and harder to steal, AND more inconvenient for the condom.

**Comment by PKPenguin321** [comment]
This PR doesn't matter since IIRC @kevinz000 made it so you can just unlock the case peacefully

**Comment by lzimann** [comment]
@KorPhaeron

**Comment by Ikarrus** [comment]
I only added the alarm in the first place for flavour and immersions, with the beneficial side effect of the AI being able to laugh at the captain.

**Comment by Shadowlight213** [comment]
@PKPenguin321 Cap's display case still requires centcom access.
https://github.com/tgstation/tgstation/blob/master/code/game/objects/structures/displaycase.dm#L243
#25793 was closed.

**Comment by optimumtact** [comment]
I dont' understand the need for this

**Pull request closed without merging**